src/main/java/com/kodilla/ecommercee/mapper/CartMapper.java

```diff
@@ -0,0 +1,46 @@
+package com.kodilla.ecommercee.mapper;
+
+import com.kodilla.ecommercee.domain.Cart;
+import com.kodilla.ecommercee.dto.CartDto;
+import com.kodilla.ecommercee.domain.Order;
+import com.kodilla.ecommercee.domain.User;
+import com.kodilla.ecommercee.service.OrderService;
```

Comment:
With this changes 
import com.kodilla.ecommercee.domain.Order;
import com.kodilla.ecommercee.domain.User;
not needed anymore...

Reply:
Removed